packages/gatsby/src/internal-plugins/query-runner/file-parser.js

```diff
@@ -83,6 +83,15 @@ async function findGraphQLTags(file, text): Promise<Array<DefinitionNode>> {
           return
         }
 
+        /**
+         * Mapping of graphql documents to concatenated location of graphql template
+         * literal that contains that document and location of graphql document inside
+         * template literal.
```

Comment:
Could we make this comment simpler to understand? Maybe this line could be split into a _What?_ and then a _How?_ section?

> A map of graphql documents to unique locations.
>
> A graphql document's unique location is made of: 
>  - the location of the graphql template literal that contains the document, and
>  - the document's location within the graphql template literal